README.md

```diff
@@ -27,26 +27,27 @@ cd /opt/climate-change-lab
 npm start
 ```
 
-The site will then be available at [http://localhost:3100](http://localhost:3100) on your host machine.
+The site will then be available at [http://localhost:4200](http://localhost:4200) on your host machine.
 
-Note that if you are using a Mac host, it may be necessary to run:
+If working within the VM, you can set `ENV.CLIMATE_CHANGE_LAB_PORT` before running `vagrant up` if port collisions for 4200 exist on your host.
+
+Note that if your host OS differs from the VM, you may need to run
 
 ```
 npm rebuild node-sass
 ```
 
-to switch between using the VM build and building directly on your host machine.
+within the environment (host or VM) before running other build scripts in that environment.
 
 ### Getting started
 
 | Command | Purpose | Use When ... |
 |------|---------|--------------|
-| `npm start` | Serve project using hot module replacement | For quick browser updating without refresh |
-| `npm run server:dev` | Serve project normally, without hot module replacement | If you don't want hot module replacement |
-| `npm run lint --silent` | Run typescript linter | To clean up your angular and .ts files |
+| `npm start` | Serve project in dev mode using Angular CLI | For quick browser updating with refresh |
+| `npm run lint` | Run typescript linter | To clean up your angular and .ts files |
```

Comment:
This list would benefit from `npm test`, though prob not `npm run e2e` until/unless we have appropriate files